selfdrive/car/toyota/interface.py

```diff
@@ -256,23 +256,17 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     # to a negative value, so it won't matter.
     ret.minEnableSpeed = -1. if (stop_and_go or ret.enableGasInterceptor) else MIN_ACC_SPEED
 
+    # on stock Toyota this is -2.5
+    ret.stopAccel = -2.5
```

Comment:
Have a route for this? Which cars does this apply to?

Reply:
Prius, this is what I've grabbed from my scanner